docs/dsp-tools-create.md

```diff
@@ -175,6 +175,101 @@ Keywords are represented as an array of strings and are used to describe and/or
 
 
 
+### Groups
+
+(optional)
+
+`"groups": [<group-definition>, <group-definition>,...]`
+
+The `groups` object contains **project specific** group definitions. As opposed to the 
+[**built-in** groups](./dsp-tools-xmlupload.md#groups), the membership of the users to the project specific groups 
+can be freely chosen by the `ProjectAdmin`. A project may define several groups such as "student-assistant", 
+"editors", etc. in order to provide their members specific permissions.
+The groups that were created here are then available in the XML file in the 
+[&lt;permissions&gt; element](dsp-tools-xmlupload.md#permissions-element).
+
+A project specific group definition has the following elements:
+
+- _name_ (mandatory): name of the group
+- _descriptions_ (mandatory): description of the group with language tags in the form `"descriptions": {"<lang>": 
+  "<string>", ...}` (currently, "en", "de", "fr", "it", and "rm" are supported)
+- _selfjoin_ (optional): true if users are allowed to join the group themselves, false (default) if a `ProjectAdmin` has 
+  to add them
+- _status_ (optional): true (default) if the group is active, false if the group is inactive
+
+Example:
+
+```json
+{
+  "groups": [
+    {
+      "name": "biz-editors",
+      "descriptions": {"en" : "Editors for the BiZ project"},
+      "selfjoin": false,
+      "status": true
+    }
+  ]
+}
+```
```

Comment:
Not sure if we should include this in the documentation as it is currently not supported by DSP-APP. Maybe we could at least add a sentence like "Groups are currently not supported by the DSP-APP. Therefore, it is discouraged to use groups."